XLS-0089d-multi-purpose-token-metadata-schema/README.md

```diff
@@ -0,0 +1,121 @@
+             <a href="mailto:smittal@ripple.com">Shashwat Mittal</a>
+             <a href="mailto:jberridi@ripple.com">Julian Berridi</a>
+
+Affiliation: <a href="https://ripple.com">Ripple</a>
+</pre>
+
+---
+
+# XLS-0089d Multi-Purpose Token Metadata Schema
+
+## 1. Abstract
+
+Multipurpose Tokens (MPTs) on the XRP Ledger allow issuers to attach arbitrary metadata to their tokens. While this flexibility is powerful, it also creates challenges for discoverability, comparability, and interoperability — both across MPTs and with existing IOUs on XRPL.
+
+This document proposes a minimally standardized metadata format for MPTs to address those challenges. The goal is not to restrict expressiveness, but to define a baseline set of fields that support reliable parsing and integration across services like block explorers, indexers, wallets, and cross-chain applications. These common fields will make it easier to surface and compare MPTs, enabling better user experiences and broader ecosystem support.
+
+This standard is optional, but MPTs that follow it will be more readily integrated into the XRPL ecosystem.
+
+---
+
+## 1.1. Approach
+
+MPTs include a 1024-byte field for arbitrary metadata. The metadata field is part of a hybrid approach of storing essential information on the ledger, while additional information can be stored off the ledger using an external URI in the metadata field. Advantages to this approach include:
+
+- Metadata can be accessed and verified on the ledger without having to fetch data from an external URI.
+- Provides increased reliability while decreasing dependency on external systems. Relying solely on external sources for metadata would introduce a dependency where services might become unavailable or unreliable, disrupting token operations and the user experience.
+- Avoids the risk of a centralized hosting service provider becoming compromised or unavailable.
+
+---
+
+## 2. Base Metadata Schema
+
+| Field             | Description                                                                                          | Example                                                                                                   | Allowed Values                                       | Type                  | Required? |
+| ----------------- | ---------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------- | ---------------------------------------------------- | --------------------- | --------- |
+| `ticker`          | Ticker symbol used to represent the token                                                            | EXMPL                                                                                                     | Any uppercase ASCII string (max 6 chars recommended) | string                | ✔️        |
```

Comment:
We should introduce a `type` string field that identifies what type of metadata this file represents. Something like `urn::mpt-meta::v1`. If omitted, this value could be implied to save space. 

This would have two benefits:

1. Primary: Will allow this specification to be versioned. If we ever want to add or remove fields from this spec, it will be nice for software to know what to expect.
2. Secondary: Will allow other MPT users to define their own metadata and give software a way to tell the difference.

Reply:
Note that I think this can be added later, per the comments above by [julian78780](https://github.com/julian78780)